compiler/AST/primitive.cpp

```diff
@@ -702,11 +702,12 @@ initPrimitive() {
   prim_def(PRIM_REQUIRE, "require", returnInfoVoid, false, false);
 }
 
-Map<const char*, VarSymbol*> memDescsMap;
+static Map<const char*, VarSymbol*> memDescsMap;
```

Comment:
This particular code does not work because, I believe, new_StringSymbol(str)->name is something like _literal_3093, and we do not want these things to show in the memory leaks table.